app/_data/tables/version_errors_konnect.yml

```diff
@@ -1709,3 +1790,87 @@ messages:
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.0.0.0` or above.
+    DocumentationURL: |
+      [Upstream entity](/gateway/latest/admin-api/#upstream-object)
+  - ID: P184
+    Severity: error
+    Description: |
+      For the `vault` entity, one or more of the following schema fields are set:
+      `config.ttl`, `config.neg_ttl`, `config.resurrect_ttl` but Kong Gateway versions
+      < 3.4.0.0 do not support these fields.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.4.0.0` or above.
+    DocumentationURL: |
+      [Vaults entity](/gateway/latest/admin-api/#vaults-object)
+  - ID: P185
+    Severity: error
+    Description: |
+      For the `rate-limiting` plugin, one or more of the following `config` fields are
+      set: `sync_rate` but Kong Gateway versions < 3.4.0.0 do not support these
+      fields. Plugin features that rely on these fields are not working as intended.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.4.0.0` or above.
+    DocumentationURL: |
+      [Rate Limiting plugin](/hub/kong-inc/rate-limiting/)
+  - ID: P186
+    Severity: error
+    Description: |
+      For the `opentelemetry` plugin, the config field `header_type` has been set to
+      `aws`. This is not supported in Kong Gateway versions < 3.4.0.0. Impacted fields
+      have been updated to their default values: `header_type=preserve`
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.4.0.0` or above.
+    DocumentationURL: |
+      [OpenTelemetry plugin](/hub/kong-inc/opentelemetry/)
+  - ID: P187
+    Severity: error
+    Description: |
+      For the `zipkin` plugin, one or more of the fields `header_type`,
+      `default_header_type` has been set to `aws`. This is not supported in Kong
+      Gateway versions < 3.4.0.0. Impacted fields have been updated to their default
+      values: `header_type=preserve` and/or `default_header_type=b3`
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.4.0.0` or above.
+    DocumentationURL: |
+      [Zipkin plugin](/hub/kong-inc/zipkin/)
+  - ID: P188
+    Severity: error
+    Description: |
+      For the `ip-restriction` plugin, the'protocols` array contains one or both of:
+      `TLS`, `TCP`.This is not supported in Kong Gateway versions < 3.4.0.0. These
+      values have been removed and plugin features that rely on these values are not
+      working as intended. If `TLS` and/or `TCP` were the only values, then protocols
+      has been reset to the default values of [`HTTP`, `HTTPS`, `GRPC`, `GRPCS`].
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.4.0.0` or above.
+    DocumentationURL: |
+      [IP Restriction plugin](/hub/kong-inc/ip-restriction/)
+  - ID: P189
+    Severity: error
+    Description: |
+      Kong Gateway version only supports the wRPC protocol which has been removed from
+      Kong Gateway >= 3.2.0.0. Konnect will be removing support for these releases of
```

Comment:
I'm confused about what this first sentence is trying to say. Is there a version number missing?

Reply:
There is! Thanks for catching :D